.github/workflows/main.yml

```diff
@@ -21,12 +21,27 @@ jobs:
         with:
           dotnet-version: '8.0.x'
 
+      - name: Add msbuild to PATH
+        if: runner.os == 'Windows'
+        uses: microsoft/setup-msbuild@v1.0.2
```

Comment:
why do we need this? can we not use `dotnet build` or `dotnet msbuild`?

Reply:
That came from the SDL2 repo and enables building C++ projects.

If there is a better way to do it, i'm happy to change it.

@harry-cpp  ?